config/navigation.txt

```diff
@@ -43,7 +43,6 @@ Learn
 
     Runtime Environment
       [/runtime/runtime]
```

Comment:
The URL like this looks a bit strange (the repeated words), could it work as `/runtime/container`, in line with the title? Though this would also need hunting down links to the page and updating those links, if any.
(all this is not problem, just comment)